release_notes.md

```diff
@@ -0,0 +1,6 @@
+### Release notes
+<!-- Please add your release notes in the following format:
+- My change description (#PR)
+ -->
+
+- Upgrade PowerShell Workers to 2.0.448 ([Release Notes](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.448))
```

Comment:
Should the link to the release notes be `https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v2.0.448`?

Reply:
Good catch, thank you.